### ISSUE TITLE
bug: `touchMove()` not firing `touchmove` event in E2E tests

### ISSUE THREAD

**Issue by nick-vincent** [ionic-team/stencil#4735]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/stencil/blob/main/CONTRIBUTING.md).
- [X] I agree to follow the [Code of Conduct](https://github.com/ionic-team/stencil/blob/main/CODE_OF_CONDUCT.md).
- [X] I have searched for [existing issues](https://github.com/ionic-team/stencil/issues) that already report this problem, without success.

### Stencil Version

4.1.0

### Current Behavior

I'm working in Stencil and trying to test that touch events are fired correctly.

I've created a barebones Stencil example repo to repro [here](https://github.com/nick-vincent/stencil-test-touch-events/).

When the [test](https://github.com/nick-vincent/stencil-test-touch-events/blob/main/src/touch-events.e2e.ts) is run, the `touchstart` and `touchend` events fire correctly, but `touchmove` does not get fired:

```ts
await page.touchscreen.touchStart(1, 1);
await page.touchscreen.touchMove(2, 2); // does not work
await page.touchscreen.touchEnd();
```

```
[50:05.8]  jest args: --spec --e2e --max-workers=8
 FAIL  src/touch-events.e2e.ts
  touch events
    ✓ fires touch start (602 ms)
    ✕ fires touch move (427 ms)
    ✓ fires touch end (381 ms)

  ● touch events › fires touch move

    expected event "touchmove" to have been called 1 times, but was called 0 time

      30 |   // touchmove does not get fired
      31 |   it('fires touch move', () => {
    > 32 |     expect(touchMoveSpy).toHaveReceivedEventTimes(1);
         |                          ^
      33 |   });
      34 |
      35 |   it('fires touch end', () => {

      at Object.<anonymous> (src/touch-events.e2e.ts:32:26)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 2 passed, 3 total
Snapshots:   0 total
Time:        2.238 s, estimated 3 s
Ran all test suites.
```

### Expected Behavior

I would expect `touchscreen.touchMove()` to fire a `touchmove` event.

### System Info

```shell
System: node 20.3.0
    Platform: darwin (22.5.0)
   CPU Model: Intel(R) Core(TM) i5-8210Y CPU @ 1.60GHz (4 cpus)
    Compiler: /node_modules/@stencil/core/compiler/stencil.js
       Build: 1692628441
     Stencil: 4.1.0 🐟
  TypeScript: 5.0.4
      Rollup: 2.42.3
      Parse5: 7.1.2
      Sizzle: 2.42.3
      Terser: 5.19.2
```

### Steps to Reproduce

```ts
import { newE2EPage, E2EPage, EventSpy } from '@stencil/core/testing';

describe('touch events', () => {
  let page: E2EPage;
  let touchStartSpy: EventSpy;
  let touchMoveSpy: EventSpy;
  let touchEndSpy: EventSpy;

  beforeEach(async () => {
    page = await newE2EPage();

    await page.setContent(`
      <html><body>Test</body></html>
    `);

    touchStartSpy = await page.spyOnEvent('touchstart');
    touchMoveSpy = await page.spyOnEvent('touchmove');
    touchEndSpy = await page.spyOnEvent('touchend');

    await page.touchscreen.touchStart(1, 1);
    await page.touchscreen.touchMove(2, 2); // does not work
    await page.touchscreen.touchEnd();
    await page.waitForChanges();
  });

  it('fires touch start', () => {
    expect(touchStartSpy).toHaveReceivedEventTimes(1);
  });

  // touchmove does not get fired
  it('fires touch move', () => {
    expect(touchMoveSpy).toHaveReceivedEventTimes(1);
  });

  it('fires touch end', () => {
    expect(touchEndSpy).toHaveReceivedEventTimes(1);
  });
});
```

### Code Reproduction URL

https://github.com/nick-vincent/stencil-test-touch-events/

### Additional Information

I thought this might be a bug with Puppeteer so I filed [an issue](https://github.com/puppeteer/puppeteer/issues/10786) there, but they asked me to file one here with Stencil.

**Comment by tanner-reits** [comment]
Hey @nick-vincent 👋 

I tested this out in a bare Node project running Puppeteer v21.1.1 and was seeing the same thing you encountered within a Stencil project. I setup some listeners to catch the events in the Puppeteer page, but the `touchstart` and `touchend` events were the only ones to emit. 

I also checked the Stencil source code and we don't handle the `touchMove()` method or `touchmove` event any differently than `touchstart` or `touchend`. I know you already reported this with the Puppeteer team and they believe it's an issue in Stencil, but it seems like this is an issue on their end.

I'm going to close this for the time-being, but if you hear anything back from their team or they can provide a vanilla puppeteer project where this is working, we'd be happy to take a further look!

**Comment by nick-vincent** [comment]
Thanks so much for the speedy response, @tanner-reits!

That doesn't surprise me — the example repo I provided doesn't even include any Stencil components, it's just the test.

I will take this back to the Puppeteer team. Wondering if you could share the barebones Node project you set up?

cc: @jrandolf

**Comment by nick-vincent** [comment]
@tanner-reits  Just to follow up here for posterity...

I did some more debugging and I was also able to repro without Stencil, so I filed [a new issue](https://github.com/puppeteer/puppeteer/issues/10836) with Puppeteer.

TL;DR: I've found that the `touchmove` event doesn't fire if the distance is less than 16px. 🤷 

I'll keep this thread updated in case other folks run into the same issue!
